.ci/release/Jenkinsfile

```
@@ -140,7 +140,7 @@ pipeline {
         stage('Nexus release') {
           steps {
             script {
-              def spid = getVaultSecret('nexus')["staging-profile-id"]
+              def spid = sh(script: "./scripts/jenkins/fetch_nexus_id.sh", returnStdout: true).trim()
```

I am not sure why this command will not work

```suggestion
              def spid = getVaultSecret('secret/apm-team/ci/nexus')["staging-profile-id"]
```

---

It will  not work because we are not running on `apm-ci`. The implementation of `getVaultSecret` depends specifically on credentials for Vault which are not deployed on the internal CI. This implementation more closely mirrors approaches which are typically employed on the internal CI.

---

Btw, I'm not objecting having the post-release job on apm-ci as long as the job that pushes to maven central remains in the internal ci. This is especially true if there's already a precedent of using secrets from vault in a public CI and if you consider that safe.